lib/messageformat.js

```diff
@@ -133,6 +133,36 @@ MessageFormat.formatters = {
   }
 };
 
+/** Predefined number formatting function enforcing Bidi Structured Text by using UCC
+ * Prevents intermingling of text contained in placeholders with the rest of message text.
+ * Maintains the integrity of Bidi text flowing according to directionality of language script.
+ * For reference to Bidi structured text see: http://cldr.unicode.org/development/development-process/design-proposals/bidi-handling-of-structured-text
+ *  @param {string} - Placeholder text content
+ *  @param {string} - Locale
+ *  @returns {string} - The resultant text fixed by UCC Bidi strong marks
+ *  @examples: (upper case stands for Bidi (Arabic/Hebrew) characters, output is shown as displayed in browser page
+ *  > var MessageFormat = require('messageformat');
+ *  > var mf = new MessageFormat('en', null, {"bidiStructuredText": MessageFormat.formatters.bidiStructuredText} );
```

Comment:
Is there a reason why the `bidiStructuredText` function needs to be user-modifiable rather than provided directly by the library? I'm essentially looking for a reason why the incantation can't be `var mf = new MessageFormat('en', null, {bidiStructuredText: true})`.